os-eap7-launch/added/launch/messaging.sh

```diff
@@ -313,14 +315,18 @@ function inject_brokers() {
       # topics environment variable name format: [NAME]_[BROKER_TYPE]_TOPICS
       topics=$(find_env "${prefix}_TOPICS")
 
+      if [ -n "${RA_TRACKING}" ]; then
```

Comment:
should be specific to the resource adapter, as in general there can be more  (should use the ${prefix}_TRACKING